_posts/2022-03-16-#24118.md

```diff
@@ -1,24 +1,72 @@
 ---
 layout: pr
 date: 2022-03-16
-title: "Add 'sweep' RPC"
+title: "Add 'sendall' RPC née sweep"
 pr: 24118
 authors: [Xekyo]
-components: ["wallet", "rpc/rest/zmq"]
+components: ["wallet", "rpc"]
```

Comment:
For future reference - the components correspond to labels on the repo, and we use them to categorize meetings (https://bitcoincore.reviews/meetings-components/) so it's best not to change them, even though this is an RPC PR.